packages/extensions-sdk/src/cli/commands/validators/check-license.ts

```diff
@@ -0,0 +1,33 @@
+import fse from 'fs-extra';
+import { type Ora } from 'ora';
+import type { Report } from '../../types.js';
+
+const checkLicense = {
+	name: 'license',
+	handler: async (spinner: Ora, reports: Array<Report>) => {
+		spinner.text = 'Check for LICENSE';
```

Comment:
Are we actually interested in the `LICENSE` file or just that the `license` field in the `package.json` is set correctly. NPM does not actually care or enforce any name for a license file and afaik we don't parse the file either, but just look at the `license` field.

Reply:
The registry data model allows for multiple registries so I was doing a catch-all. This can always be changed to a warning as the point is improving the quality and standards of extensions surfaced to users?

Reply:
We should then additionally check the `license` field in the package.json, IMO since that is the field we actually display to the user, as far as I know.

Reply:
We should _only_ check the `license` field 😃👍